features/node.yml

```diff
@@ -0,0 +1,82 @@
+name: Element / Node
```

Comment:
Yeah, this all feels DOM-ish to me. It feels a little weird to talk about elements and nodes with acknowledging that they're the branches and leaves of the tree.